app/pages/planner/page.tsx

```diff
@@ -198,6 +285,19 @@ export default function Planner() {
                       />
                     )}
                   </div>
+                  <div className="text-2xl font-bold">Selected Courses</div>
+                  <div>
+                    <button
+                      className="text-white bg-blue-600 hover:bg-blue-800 font-medium rounded-lg text-base px-2 py-1"
+                      type="button"
+                      onClick={() => {
+                        handleCourseSaving();
+                      }}
```

Comment:
The inner set of braces here is redundant, right? (If we’re using optional braces everywhere, just ignore and resolve!)